<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>345</width>
    <height>130</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>解方程软件by娄卫健</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
QWidget{background-color:rgb(29, 28, 28);
color:rgb(116, 116, 116)
}
QLineEdit{background-color:rgb(19, 19, 19);
color:rgb(214, 214, 214);
border:none;}
QTabWidget{border-radius:25px;
background-color:rgb(29, 28, 28);}
QPushButton{background-color:rgb(29, 28, 28);
color:rgb(254, 118, 26);
border-radius:25px;
border:none;
}</string>
  </property>
  <widget class="QTabWidget" name="tab0">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>541</width>
     <height>401</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>幼圆</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="tab11">
    <attribute name="title">
     <string>一元方程</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>101</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入方程式</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button_2">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>-20</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>解方程</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab22">
    <attribute name="title">
     <string>二元方程</string>
    </attribute>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>50</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入方程式2</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>231</width>
       <height>51</height>
      </rect>
     </property>
     <property name="placeholderText">
      <string>请输入方程式1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="button">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>-20</y>
       <width>121</width>
       <height>121</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>幼圆</family>
       <pointsize>18</pointsize>
      </font>
     </property>
     <property name="text">
      <string>解方程</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>菜单</string>
    </attribute>
    <widget class="QPushButton" name="btmenu">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>-20</y>
       <width>331</width>
       <height>111</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton {background-image:url(back4.png);
color: rgb(255,121,32) ;}</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>tab0</tabstop>
  <tabstop>lineEdit</tabstop>
  <tabstop>button</tabstop>
  <tabstop>lineEdit_1</tabstop>
  <tabstop>lineEdit_2</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
